libs/sdk-ui-semantic-search/config/rush-project.json

```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush-project.schema.json",
+    "incrementalBuildIgnoredGlobs": [],
+    "operationSettings": [
+        {
+            "operationName": "build",
+            "outputFolderNames": ["esm"]
```

Comment:
I guess that styles/scss is missing in rush cache definition see: https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-kit/config/rush-project.json#L7